ETOAST-115: Tag component

suabochica opened the pull request:
For more detaisl please check the JIRA ticket [ETOAST-115](https://jira.endava.com/browse/ETOAST-115).

suabochica committed:
ETOAST-115: Tag initial commit

suabochica committed:
E-TOAST-115: Add base of the tag component

suabochica committed:
E-TOAST-115: Tag component implementation

Jkierem commented on src/Tag/index.tsx:
```diff
@@ -0,0 +1,62 @@
+import getClassName from 'getclassname';
+
+interface TagProps {
+  /**
+   * The `close` attribute to be passed to the create a tag element with close action
+   */
+  hasCloseAction: boolean;
```
que tal si cambiamos esto por un callback opcional que si es verdadero se ejecute cuando le den click a la "X"?

john-angel-endava committed:
E-TOAST-116: Select component

john-angel-endava committed:
Select and Options containers

john-angel-endava committed:
use div instead of ul for select options
- hide options when one is selected

john-angel-endava committed:
change select options type to array

john-angel-endava committed:
Add span elements to use Select selectors defined in @e-toast/css
Control Select arrow icon style via useEffect
Document Select props and local parameters

john-angel-endava committed:
Select properties updated:
add onChange event
set option selected as default

john-angel-endava committed:
add disable property. use disable class on select elements

john-angel-endava committed:
refactor content class definition on select

john-angel-endava committed:
refactor select icon class control. update select README.md

john-angel-endava committed:
E-TOAST 116: Select component

john-angel-endava committed:
add selected prop to control default option to be displayed on select

john-angel-endava committed:
E-Toast 116: refactor select elements classes control with getClassName

john-angel-endava committed:
fix Indentation space mix on SelectProps interface

john-angel-endava committed:
Merge branch 'feature/select' of https://github.com/bod-endava/e-toast-react into feature/select required after squash

Jkierem committed:
ETOAST-117: Form Component (#4)

suabochica committed:
Merge branch 'feature/Tag' of github.com:bod-endava/e-toast-react into feature/Tag
